app/views/ProfileSetupView.scala.html

```diff
@@ -0,0 +1,54 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+    layout: templates.Layout,
+    govukButton: GovukButton
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@layout(
+    pageTitle    = titleNoForm(messages("profileSetup.title")),
+    showBackLink = false
+) {
+
+    <h1 class="govuk-heading-l">@messages("profileSetup.h1")</h1>
+
+    <p class="govuk-body">@messages("profileSetup.body.intro")</p>
+
+    <h2 class="govuk-heading-m">@messages("profileSetup.ukims.h2")</h2>
+    <p class="govuk-body">@messages("profileSetup.p1")</p>
+    <p class="govuk-body">@messages("profileSetup.p2")</p>
+    <p class="govuk-body">@messages("profileSetup.p3") <a href="https://www.gov.uk/guidance/apply-for-authorisation-for-the-uk-internal-market-scheme-if-you-bring-goods-into-northern-ireland">@messages("profileSetup.p3.linkText")</a>@messages("site.fullStop")</p>
```

Comment:
Do we have any rules for linting - for instance extra long lines should be split?

Reply:
There's a default scalafmt file for that :) We can modify it if we're not happy with the default

Reply:
This is refactored now